### PR TITLE
Make it easier to find workspace id

### PR DESCRIPTION
**Pull request by omerlh**

### CODE REVIEW

File: docs/attach-monitoring-tags.md [hunk 1]
```diff
@@ -51,11 +51,13 @@ The configuration change can take a few minutes to complete. When it finishes, y
 # login
 az login
 
+workspace="/subscriptions/<subscription id>/resourceGroups/<resource group name>/providers/Microsoft.OperationalInsights/workspaces/<workspace name>"
```
[the remark this answers is not in the document]
Comment (omerlh) [reply docs/attach-monitoring-tags.md]: Yep, fixed

[the remark this answers is not in the document]
Comment (ganga1980) [reply docs/attach-monitoring-tags.md]: Thanks for making these changes. Btw, are you planning the monitoring  for acs-engine clusters with this new experience https://aka.ms/azmon-containers? If yes, please let me know if you have any feedback.

[the remark this answers is not in the document]
Comment (omerlh) [reply docs/attach-monitoring-tags.md]: I'm actually evaluating it right now - but we're out of space in the workspace account, so I'll have to wait until tomorrow to see it in action. This is how I got to this document in the first place

[the remark this answers is not in the document]
Comment (ganga1980) [reply docs/attach-monitoring-tags.md]: Got it. Please feel free to ping me if you need any additional details or have feedback.

**Pull request merged**